Sources/PayPalUI/PayPalPayLaterButton.swift

```diff
@@ -41,29 +38,75 @@ public final class PayPalPayLaterButton: PaymentButton, UIViewRepresentable {
             insets: insets,
             label: .payLater
         )
-        self.action = action
     }
+}
+
+public extension PayPalPayLaterButton {
+    
+    /// PayPalPayLaterButton for SwiftUI
+    struct Representable: UIViewRepresentable {
+        
+        private let button: PayPalPayLaterButton
+        private var action: () -> Void = { }
+        
+        /// Initialize a PayPalPayLaterButton
+        /// - Parameters:
+        ///   - insets: Edge insets of the button, defining the spacing of the button's edges relative to its content.
+        ///   - color: Color of the button. Default to gold if not provided.
+        ///   - edges: Edges of the button. Default to softEdges if not provided.
+        ///   - size: Size of the button. Default to collapsed if not provided.
+        public init(
+            insets: NSDirectionalEdgeInsets? = nil,
+            color: PayPalPayLaterButton.Color = .gold,
+            edges: PaymentButtonEdges = .softEdges,
+            size: PaymentButtonSize = .collapsed,
+            _ action: @escaping () -> Void = { }
+        ) {
+            self.button = PayPalPayLaterButton(
+                fundingSource: .payLater,
+                color: color.color,
+                edges: edges,
+                size: size,
+                insets: insets,
+                label: .payLater
+            )
+            self.action = action
+        }
+
+        // MARK: - UIViewRepresentable methods
```

Comment:
Can we leave a TODO + create a JIRA for unit testing the rest of the `UIViewRepresentable` implementation?